optd-datafusion-repr/src/explain.rs

```diff
@@ -0,0 +1,15 @@
+use optd_core::rel_node::RelNodeMeta;
+use pretty_xmlish::Pretty;
+
+use crate::cost::ROW_COUNT;
+
+pub trait Insertable<'a> {
+    fn with_meta(self, meta: &RelNodeMeta) -> Self;
+}
+
+impl<'a> Insertable<'a> for Vec<(&'a str, Pretty<'a>)> {
+    fn with_meta(mut self, meta: &RelNodeMeta) -> Self {
+        self.push(("rows", Pretty::display(&meta.cost.0[ROW_COUNT])));
```

Comment:
I would recommend maybe using `CostModel::explain`.

This would require to modify Cost model's signature from 
`fn explain(&self, cost: &Cost) -> String` to `fn explain(cost: &Cost) -> String`

```suggestion
        self.push("cost", Pretty::display(CostModel::explain(&meta.cost)));
```